gapil/compiler/map.go

```diff
@@ -413,5 +413,6 @@ func (c *compiler) buildMapType(t *semantic.Map) {
 	c.ty.maps[t] = mi
 }
 
-const mapGrowBy = 16
+const mapGrowMultiplier = 2
```

Comment:
If we're implementing the map in C++ as well, it would be really nice if this could be declared in `builtins.h` and reused